### PR TITLE
Instructions on how to create your niteo career repo

### PR DESCRIPTION
**Pull request by zupo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
 # My Niteo Career
-My [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1) and [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2).
+
+Every Nitean creates a repository like this one in their own GitHub account. The reason we do it like this is that the repository might contain personal information and we want every Nitean to be the master of their personal information. If they leave Niteo, they revoke access to their career repo and noone can read it any more. It also gives Niteans clarity into who exactly can read the One on One's notes: only the people listed as collaborators to the repo.
+
+The "meat" of the repo are two issues:
```
Comment (dz0ny) [inline README.md]:
```suggestion
The "gist" of the repo are two issues:
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Every Nitean creates a repository like this one in their own GitHub account. The reason we do it like this is that the repository might contain personal information and we want every Nitean to be the master of their personal information. If they leave Niteo, they revoke access to their career repo and noone can read it any more. It also gives Niteans clarity into who exactly can read the One on One's notes: only the people listed as collaborators to the repo.
 
-The "meat" of the repo are two issues:
+The gist of the repo are two issues:
```
Comment (zupo) [inline README.md]: Not an improvement. A gist is a summary. And it's not what I meant. I meant "the main part", but didn't want to use such a dull expression. 

The _bulk_ of the repo, maybe?

Comment (dmurko) [reply README.md]: `There are two issues in the repository:`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
 # My Niteo Career
-My [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1) and [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2).
+
+Every Nitean creates a repository like this one in their own GitHub account. The reason we do it like this is that the repository might contain personal information and we want every Nitean to be the master of their personal information. If they leave Niteo, they revoke access to their career repo and noone can read it any more. It also gives Niteans clarity into who exactly can read the One on One's notes: only the people listed as collaborators to the repo.
```
Comment (marbeann) [inline README.md]: - nonone -> no one

- One on One's notes -> One on One note

File: README.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+
+The "meat" of the repo are two issues:
+* [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1): Every time you have a One on One. you or your senior writes a report here. The issues provides a template for the report.
```
Comment (marbeann) [inline README.md]: One. you -> comma instead of full stop

File: README.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
...
+* [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2): Every time your have a One on One, you talk about the weeks since the last One on One and highlight potentials wins in this issue and recognize the steps required to get promoted to the next level.
```
Comment (marbeann) [inline README.md]: _...time your have..._ - > *you have

File: README.md [hunk 6]
```diff
@@ -1,2 +1,30 @@
...
+* [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1): Every time you have a One on One. you or your senior writes a report here. The issues provides a template for the report.
+* [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2): Every time your have a One on One, you talk about the weeks since the last One on One and highlight potentials wins in this issue and recognize the steps required to get promoted to the next level. 
+
+## How to create the Niteo Career repo for yourself
+
+### Create the repo
+
+1. Go to https://import.github.com/.
+1. For `Your old repository’s clone URL` enter `https://github.com/niteoweb/my-niteo-career.git`.
+1. For `Name` enter `my-niteo-career`.
```
Comment (marbeann) [inline README.md]: Since we're writing instructions, isn't "now" the best time to suggest a different name? "Career" perhaps.

**Comment by zupo** [comment]
@marbeann, @dmurko: ready for re-review

File: .github/ISSUE_TEMPLATE/catchups.md [hunk 7]
```diff
@@ -17,26 +17,26 @@ A few questions to cover on One on Ones:
...
-* improving managing projects  
+* improving managing projects
 * chat
 
 ## Action Items
 
 Nitean will:
 
-* write his Nitean profile 
+* write his Nitean profile
```
Comment (marbeann) [inline .github/ISSUE_TEMPLATE/catchups.md]: his - > their (Singular they)

Comment (zupo) [reply .github/ISSUE_TEMPLATE/catchups.md]: tnx!

**Pull request merged**